test/testutils/util.go

```diff
@@ -35,8 +35,18 @@ import (
 	acceleratorutils "sigs.k8s.io/lws/pkg/utils/accelerators"
 )
 
+func CreateLws(ctx context.Context, k8sClient client.Client, lws *leaderworkerset.LeaderWorkerSet) {
+	Eventually(k8sClient.Create(ctx, lws)).Should(Succeed())
+	Eventually(func() error {
```

Comment:
Maybe we should expect lws is available here? If we create successfully, I can't image why we'll not get the object here unless the apiserver/etcd is down.